cmd/memberagent/main.go

```diff
@@ -160,8 +160,15 @@ func buildHubConfig(hubURL string, useCAAuth bool, tlsClientInsecure bool) (*res
 
 	hubConfig.TLSClientConfig.Insecure = tlsClientInsecure
 	if !tlsClientInsecure {
-		hubConfig.TLSClientConfig.CAFile = os.Getenv("CA_BUNDLE")
+		caBundle := os.Getenv("CA_BUNDLE")
```

Comment:
```suggestion
		caBundle, err := os.LookupEnv("CA_BUNDLE")
```

Reply:
in this way, we can differentiate when user didn't set the env vs set it as empty. Empty is not allowed.

Reply:
I have fixed that in [this commit](https://github.com/Azure/fleet/pull/365/commits/52b9aaa882e5156dd0dd6ef7326492be4f8677c2).

I also checked all other places are using `os.Getenv()`. Other places don't need change since there are empty string  checking for all of them.

Reply:
yes, this is only needed when there is  a reasonable "default" value.